src/ElectricPokemon.java

```diff
@@ -0,0 +1,73 @@
+public class ElectricPokemon extends Pokemon {
+    private boolean glowsInTheDark;
+    private int numberOfColors;
+    private boolean largerThenAMeter;
+    private boolean fightsInGroupOnly;
+
+    public ElectricPokemon(String name, int level, String hp, String food, String sound, String type, boolean glowsInTheDark, int numberOfColors, boolean largerThenAMeter, boolean fightsInGroupOnly) {
+        super(name, level, hp, food, sound, type);
+        this.glowsInTheDark = glowsInTheDark;
+        this.numberOfColors = numberOfColors;
+        this.largerThenAMeter = largerThenAMeter;
+        this.fightsInGroupOnly = fightsInGroupOnly;
+    }
+
+    public boolean isGlowsInTheDark() {
+        return glowsInTheDark;
+    }
+
+    public void setGlowsInTheDark(boolean glowsInTheDark) {
+        this.glowsInTheDark = glowsInTheDark;
+    }
+
+    public int getNumberOfColors() {
+        return numberOfColors;
+    }
+
+    public void setNumberOfColors(int numberOfColors) {
+        this.numberOfColors = numberOfColors;
+    }
+
+    public boolean isLargerThenAMeter() {
```

Comment:
Goed dat je in deze functie "is" gebruikt in plaats van "get". Het is conventie in Java om voor boolean getters "is" te gebruiken.